packages/lambdas/src/hl7-lahie-sftp-ingestion.ts

```diff
@@ -0,0 +1,20 @@
+import { capture } from "./shared/capture";
+import { prefixedLog } from "./shared/log";
+import { getEnvVarOrFail } from "@metriport/shared/common/env-var";
+import { Hl7SubscriptionLahieIngestionDirect } from "@metriport/core/command/hl7-sftp-ingestion/hl7-sftp-ingestion-direct";
+import { SftpIngestionClient } from "@metriport/core/command/hl7-sftp-ingestion/sftp-ingestion-client";
+
+capture.init();
+
+// Automatically set by AWS
+const lambdaName = getEnvVarOrFail("AWS_LAMBDA_FUNCTION_NAME");
+
+export const handler = capture.wrapHandler(async () => {
+  capture.setExtra({ context: lambdaName });
+  const log = prefixedLog("Lahie-ingestion");
+  log("Starting ingestion of Lahie ADTs");
+  const sftpClient = await SftpIngestionClient.create(log);
+  const handler = new Hl7SubscriptionLahieIngestionDirect(sftpClient, log);
```

Comment:
The ingestino client should go inside the direct lambda implementation. We want the cloud handler to be _as_ minimal as possible of a wrapper around the direct implementation to minimize environment differences.